src/util/selectorAtCursor.js

```diff
@@ -0,0 +1,27 @@
+import postcss from 'postcss';
+
+export function selectorAtCursor(source, cursor, language) {
+  let highlighterSelector = '';
```

Comment:
Echoing an earlier comment seems like this should be `null` if we don’t find a selector.